### PR TITLE
Update copy changes for call advisor section

### PR DESCRIPTION
**Pull request by tavy88**
### Context

Copy changes for:

- call advisor section
- next steps section
- courses page section

### Screenhsots

Screen Shot 2019-08-08 at 17 25 27

### JIRA

https://dfedigital.atlassian.net/browse/GET-318

### CODE REVIEW

File: app/views/pages/task_list.html.erb [hunk 1]
```diff
@@ -13,7 +13,7 @@
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-xl"><%= t('.title') %></h1>
     <p class="govuk-!-margin-bottom-9 govuk-body-m">
-      Check what you need to do to retrain for another job. How long it takes you to complete the following steps will depend on your situation.
+      Here's what you need to do to retrain for another job.
```
Comment (cwrw) [inline app/views/pages/task_list.html.erb]: is this part of this ticket, I think this is part of another ticket?

File: app/views/courses/index.html.erb [hunk 2]
```diff
@@ -14,6 +14,7 @@
 <div class="govuk-grid-row govuk-!-margin-top-7">
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-xl"><%= params[:topic_id].titleize %> courses near me</h1>
+    <p class="govuk-body-l">Apply for a course by calling the course provider or visiting their website.</p>
```
Comment (cwrw) [inline app/views/courses/index.html.erb]: same here regarding not part of this ticket

File: app/views/pages/task_list.html.erb (hunk 1 above)
Comment (tavy88) [reply app/views/pages/task_list.html.erb]: Will clarify with Steve F. It's in review to him atm.

File: app/views/courses/index.html.erb (hunk 2 above)
Comment (tavy88) [reply app/views/courses/index.html.erb]: Will clarify with Steve F. It's in review to him atm.

File: app/views/pages/task_list.html.erb (hunk 1 above)
Comment (tavy88) [reply app/views/pages/task_list.html.erb]: Confirmed with him it's a valid change.

File: app/views/courses/index.html.erb (hunk 2 above)
Comment (tavy88) [reply app/views/courses/index.html.erb]: Confirmed with him it's a valid change.

**Pull request merged**